apps/api/src/scraper/WebScraper/sitemap.ts

```diff
@@ -36,17 +36,15 @@ export async function getLinksFromSitemap(
     const root = parsed.urlset || parsed.sitemapindex;
 
     if (root && root.sitemap) {
-      for (const sitemap of root.sitemap) {
-        if (sitemap.loc && sitemap.loc.length > 0) {
-          await getLinksFromSitemap({ sitemapUrl: sitemap.loc[0], allUrls, mode });
-        }
-      }
+      const sitemapPromises = root.sitemap
```

Comment:
ok I think this may be part of another PR. but ok, let's merge it and I can use this to improve the crawl sitemap thing